src/Helper.cs

```diff
@@ -0,0 +1,18 @@
+using System.Text;
+
+namespace BIP39Wallet
+{
+    public static class Helper
```

Comment:
If we don't intend to expose this class, make it internal.

Reply:
made it internal